Tests/conf.json

```diff
@@ -908,6 +908,13 @@
             "fromversion": "5.5.0",
             "timeout": 1600
         },
+        {
+            "integrations": "Panorama",
+            "instance_names": "palo_alto_panorama",
```

Comment:
if these commands are available on the FW as well, please also add an entry on the `palo_alto_firewall` or `palo_alto_firewall_9.0`